frontend/src/pages/LegacyReport/__tests__/index.js

```diff
@@ -49,6 +56,36 @@ describe('LegacyReport', () => {
     expect(alert).toHaveTextContent('Unable to load activity report');
   });
 
```

Comment:
These tests assert that 
1) an admin sees the button
2) clicking the button toggles the form
3) clicking the save button sends a put request

There is also a test for if the put request bombs